docs/getting-started/integrations-cdn/vuejs-v2.md

```diff
@@ -0,0 +1,147 @@
+---
+menu-title: Vue.js 2.x
+meta-title: Vue.js 2.x rich text editor component with CDN | CKEditor 5 documentation
+meta-description: Install, integrate and configure CKEditor 5 using the Vue.js 2.x component with CDN.
+category: cloud
+order: 40
+---
+
+{@snippet installation/integrations/framework-integration}
+
+# Vue.js 2.x rich text editor component
+
+<p>
+	<a href="https://www.npmjs.com/package/@ckeditor/ckeditor5-vue2" target="_blank" rel="noopener">
+		<img src="https://badge.fury.io/js/%40ckeditor%2Fckeditor5-vue2.svg" alt="npm version" loading="lazy">
+	</a>
+</p>
+
+<info-box warning>
+	This guide is about the CKEditor&nbsp;5 integration with Vue.js 2.x. However, Vue 2 has reached EOL and is no longer actively maintained. To learn more about the integration with Vue.js 3+, check out the {@link getting-started/integrations-cdn/vuejs-v3 "Rich text editor component for Vue.js 3+"} guide.
+</info-box>
+
+Vue.js is a versatile framework for building web user interfaces. CKEditor&nbsp;5 provides the official Vue component you can use in your application. This guide assumes that you already have a Vue 2 project.
+
+{@snippet getting-started/use-builder}
+
+## Quick start
+
+First, install the CKEditor&nbsp;5 WYSIWYG editor component for Vue 2:
+
+```bash
+npm install @ckeditor/ckeditor5-vue2
+```
+
+Then, include the CKEditor&nbsp;5 scripts and styles. All necessary scripts and links are in the HTML snippet below. You can copy and paste them into your `index.html`. Open-source and premium features are in separate files, so there are different tags for both types of plugins. Add tags for premium features only if you use them.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" href="/favicon.ico" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		
+		<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.css" />
+		<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.umd.js"></script>
+		<!-- Add if you use premium features -->
+        <link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5-premium-features/{@var ckeditor5-version}/ckeditor5-premium-features.css" />
+        <script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5-premium-features.umd.js"></script>
+		<!--  -->
+
+		<title>Vite App</title>
```

Comment:
Vue?

Reply:
It's a bundler-generated title, which is Vite. But, sure - I'll change it.

Reply:
Hmm... It is a good point. On the other hand - it doesn't change much. So, I updated this detail.